mojo/stdlib/test/collections/test_list_getitem_invalid_index_int.mojo

```diff
@@ -23,5 +25,8 @@ fn test_fail_list_index():
     print("is never reached")
 
 
-fn main():
-    test_fail_list_index()
+def main():
```

Comment:
This file is a bit special as it actually tests that the resulting executable asserts and exits early. We should probably have some sort of way to test that a function "fails" instead of succeeds. For now though, files like these probably shouldn't use `TestSuite`.